.github/workflows/regenerate.yaml

```diff
@@ -55,10 +55,7 @@ jobs:
       - run: yarn install --frozen-lockfile
 
       - name: Generate commit_hash.sh
-        run: echo $'COMMIT_HASH=${COMMIT_HASH}\n' >> downloaded/commit_hash.sh
-        shell: bash
-        env:
-          COMMIT_HASH: ${{ github.event.inputs.openapi_grpc_sha }}
+        run: echo $'COMMIT_HASH=${{ github.event.inputs.openapi_grpc_sha }}\n' > ./downloaded/commit_hash.sh
```

Comment:
just for my education, what does the leading $ do in `echo $'`?

Reply:
handles `\n` as a newline. Without it, the character `n` was being emitted after the sha value